pubtools/iib/iib_ops.py

```diff
@@ -205,8 +219,21 @@ def _iib_op_main(args, operation=None, items_final_state="PUSHED"):
         extra_args = {"cnr_token": args.iib_cnr_token}
         if args.iib_legacy_org:
             extra_args["organization"] = args.iib_legacy_org
+
     if args.overwrite_from_index:
-        extra_args["overwrite_from_index"] = args.overwrite_from_index
+        if args.overwrite_from_index_token:
+            extra_args["overwrite_from_index"] = args.overwrite_from_index
+            extra_args["overwrite_from_index_token"] = args.overwrite_from_index_token
+        else:
+            raise ValueError(
+                "Either both or neither of overwrite-from-index and "
+                "overwrite-from-index-token should be specified."
+            )
+    elif args.overwrite_from_index_token:
```

Comment:
Same as for iiblib. I don't think it makes sense to do these checks on the client side. Plus, when you already suggested checking this in iiblib it doesn't make sense to check it also here.